cfgov/jinja2/v1/_includes/atoms/hyperlink.html

```diff
@@ -15,4 +15,4 @@
    value.text:    A string for the text of the link.
 
    ========================================================================== #}
-<a class="jump-link" href="{{ value.url }}">{{ value.text }}</a>
\ No newline at end of file
+<a class="a-link__jump" href="{{ value.url }}">{{ value.text }}</a>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/2cfd1760b2e7df96cce4ec492d0ced7a2847f39e